plugins/woocommerce-blocks/package.json

```diff
@@ -50,7 +50,7 @@
 		"build:project": "pnpm --if-present '/^build:project:.*$/'",
 		"build:project:bundle": "wireit",
 		"build:check-assets": "rimraf build/* && cross-env ASSET_CHECK=true BABEL_ENV=default NODE_ENV=production webpack",
-		"build:deploy": "rimraf vendor/* && cross-env WOOCOMMERCE_BLOCKS_PHASE=2 composer install --no-dev && cross-env WOOCOMMERCE_BLOCKS_PHASE=2 pnpm run build --loglevel error",
+		"build:deploy": "rimraf vendor/* && cross-env composer install --no-dev && cross-env pnpm run build --loglevel error",
```

Comment:
I don't think this is used anymore either? The top level commands only build `build:project:.*`

Reply:
Removed this